### PR TITLE
Update release_team.md

### PR DESCRIPTION
**Pull request by jdumars**
This is the initial and incomplete release team roster.  Subsequent PRs will be added as more volunteers are identified.

### CODE REVIEW

File: releases/release-1.10/release_team.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 | Branch Manager | | |
-| Docs | | |
-| Release Notes | | |
-| Marketing | | |
-| Patch Release Manager | | |
+| Docs | Jennifer Rondeau ([@Bradamant3](https://github.com/Bradamant3)) | Jessica Yao ([@abiogenesis-now](https://github.com/abiogenesis-now)) |
+| Release Notes | Nick Chase ([@nickchase](https://github.com/nickchase)) | |
+| Marketing | Caitlyn O'Connell | |
+| Patch Release Manager | Shyam JVS ([@shyamjvs](https://github.com/shyamjvs)) | |
+| Approval Notifier | 	k8s-ci-robot ([@k8s-ci-robot](https://github.com/k8s-ci-robot)) | |
```
Comment (enisoc) [inline releases/release-1.10/release_team.md]: This one was a joke. You don't have to include it in the final list. :)

Comment (jdumars) [reply releases/release-1.10/release_team.md]: Bots never get enough recognition!  ;)  It's staying.

Comment (idvoretskyi) [reply releases/release-1.10/release_team.md]: This guy should be recognized :)

**Pull request merged**